packages/react-core/src/helpers/util.ts

```diff
@@ -318,7 +318,7 @@ export const formatBreakpointMods = (
  */
 export const getBreakpoint = (width: number): 'default' | 'sm' | 'md' | 'lg' | 'xl' | '2xl' => {
   if (width === null) {
-    width = canUseDOM ? window.innerWidth : 1200;
+    return null;
```

Comment:
This does cause a classname to get applied - `pf-m-breakpoint-null` so I feel like that's not quite ideal?

Reply:
thanks, I added a check in the page component to make sure width is defined first